packages/infra/lib/api-stack.ts

```diff
@@ -97,6 +98,75 @@ export class APIStack extends Stack {
       slackNotification?.alarmAction
     );
 
+    //-------------------------------------------
+    // Application-wide feature flags
+    //-------------------------------------------
```

Comment:
Can we move this to a separated file? This is super long already. NABD if we're planning to do it in the next few days, IMO.

Reply:
done